src/python/grpcio/grpc/_auth.py

```diff
@@ -60,15 +69,7 @@ def __call__(self, context, callback):
         else:
             future = self._pool.submit(self._credentials.get_access_token)
         future.add_done_callback(
-            lambda x: self._get_token_callback(callback, x))
-
-    def _get_token_callback(self, callback, future):
-        try:
-            access_token = future.result().access_token
-        except Exception as e:
-            _sign_request(callback, None, e)
-        else:
-            _sign_request(callback, access_token, None)
+            lambda ignored_future: _get_token_callback(callback, future))
```

Comment:
So the renaming of `x` is because [the style guide forbids single-letter identifiers](https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming).

The preference for `future` over `ignored_future` comes from a practice of using a value by the earliest name to which it was bound among all available names for the value (this comes up so rarely that I'm not sure I've ever thought it through in these words before...). The indication to the reader of the used future being the same future bound in the enclosing scope is worth whatever dynamic penalty is incurred (if it's even present and measureable rather than optimized away?).

Also I like to feel like I'm rebelling against the `Future.add_done_callback` API which I don't think should have been designed to pass itself to the callbacks registered with it. :stuck_out_tongue: